package.json

```diff
@@ -47,6 +47,7 @@
     "mockery": "~2.1.0",
     "nock": "^10.0.6",
     "nyc": "^14.1.1",
+    "sinon": "^11.1.2",
```

Comment:
I think that adding sinon just for one test is an overkill. We can achieve the same with using mockery, which we're using already in the cli tests quite extensively.

Reply:
sure will check how do we do it in mockery, however this is just in the devs dependencies

Reply:
I realize that, I'm just saying that it's easier to use the same mocking library that we already use across the project instead of adding a new one for just one test. Unless we plan to replace mockery with sinon of course.

Reply:
https://www.tabnine.com/code/javascript/modules/mockery
its mentioned over here to stub using sinon and mock using mockery